metricbeat/module/system/diskio/diskstat_linux.go

```diff
@@ -32,6 +32,12 @@ func Get_CLK_TCK() uint32 {
 	return uint32(100)
 }
 
+// function should map functionality to disk package for linux os
+func IOCounters(names ...string) (map[string]disk.IOCountersStat, error) {
+	stats, err := disk.IOCounters(names...)
```

Comment:
Looks like no need to assign the variables first and you can directly use `return := ...` here.